### PR TITLE
D1234

### PR DESCRIPTION
**Pull request by valerii-fed**
PR body

### CODE REVIEW

**Comment by valerii-fed** [comment]
Created by: **valerii.fedorenko.ua@ukr.net**
Time: **Thu Sep 27 22:27:15 EEST 2018**
Created by: **valerii.fedorenko.ua@gmail.com**
Time: **Thu Sep 27 22:26:44 EEST 2018**
PR comment

**Comment by val-fed** [comment]
new PR comment

File: test.txt [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
-test5
+test2test5
 test6
-test7
 test8
-test9
+test999999999999999999
 test10
 test11
 test12
```
Comment (val-fed) [inline test.txt]: line 8

File: test.txt [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
-test17
 test18
 test19
-test20
+20
 test21
-
+D1234
+test15
+test16
```
Comment (val-fed) [inline test.txt]: line 21

**Pull request closed without merging**